src/main/java/ch/ivyteam/ivy/maven/engine/EngineControl.java

```diff
@@ -198,13 +200,41 @@ private void findStartEngineUrl(String newLine)
     if (newLine.contains("info page of Axon.ivy Engine") && !engineStarted.get())
     {
       String url = StringUtils.substringBetween(newLine, "http://", "/");
-      url = "http://" + url + "/ivy/";
+      url = "http://" + url + "/";
+      url += evaluateDefaultContext(url);
       context.log.info("Axon.ivy Engine runs on : " + url);
       context.properties.setMavenProperty(Property.TEST_ENGINE_URL, url);
       engineStarted.set(true);
     }
   }
 
+  private String evaluateDefaultContext(String url)
+  {
+    HttpURLConnection connection = null;
+    try
+    {
+      connection = (HttpURLConnection) new URL(url).openConnection();
```

Comment:
Would be nice to use the new approach!
https://golb.hplar.ch/2019/01/java-11-http-client.html